src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -10785,6 +10802,166 @@ namespace OutputReportTabular {
 		}
 	}
 
+    // Parses the contents of the EIO (initializations) file and creates subtables for each type of record in the tabular output files
+	// Glazer - November 2016
+	void
+	WriteEioTables() {
+
+		if (displayEioSummary)
+		{
+			// all arrays are in the format: (row, column)
+			Array1D_string columnHead;
+			Array1D_int columnWidth;
+			Array1D_string rowHead;
+			Array2D_string tableBody;
+			Array1D_int colUnitConv;
+
+			// setting up  report header 
+			WriteReportHeaders("Initialization Summary", "Entire Facility", isAverage);
+
+			// since the EIO initilization file is open at this point must close it to read it and then reopen afterward.
+			gio::close(OutputFileInits); 		// for testing only
+
+			std::ifstream eioFile;
```

Comment:
Frack.  OK, I see now.  First question: How did you come up with the 822?  I get 724:

```
$ grep -r 'gio::write( OutputFileInits' * | wc -l
     724
```

I don't ask this to correct your number, I ask because I'm imagining setting up a script to convert the lines into function calls.  It would be nice to convert these lines:

```
gio::write( OutputFileInits, fmtA ) << "! <Site:GroundTemperature:BuildingSurface>, Months From Jan to Dec {C}";
```

into:

```
writeToEIO( formatString, args... );
```

possibly using variadics.  But ugh.  I don't think I want to go down that path right now.  I would love to see a more C++ approach to IO brought in, but that isn't a part of this PR.  Can you just respond to the other changes I mentioned and we'll move to get this merged.